lib/aws/bootstrap/tf-providers-aws.j2.tf

```diff
@@ -14,7 +14,7 @@ terraform {
     }
     helm = {
       source = "hashicorp/helm"
-      version = "~> 1.3.2"
+      version = "~> 2.1.2"
```

Comment:
please remove this as we're going to remove it shortly

Reply:
done